### PR TITLE
More Windows build fixes

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: libraries/networking/src/LimitedNodeList.cpp [hunk 1]
```diff
@@ -95,8 +95,8 @@ LimitedNodeList::LimitedNodeList(unsigned short socketListenPort, unsigned short
     
     // set &PacketReceiver::handleVerifiedPacket as the verified packet callback for the udt::Socket
     using std::placeholders::_1;
-    _nodeSocket.setPacketHandler(std::bind(&PacketReceiver::handleVerifiedPacket, _packetReceiver, _1));
-    _nodeSocket.setPacketListHandler(std::bind(&PacketReceiver::handleVerifiedPacketList, _packetReceiver, _1));
+    _nodeSocket.setPacketHandler([this](std::unique_ptr<udt::Packet> packet) { _packetReceiver->handleVerifiedPacket(std::move(packet)); });
+    _nodeSocket.setPacketListHandler([this](std::unique_ptr<udt::PacketList> packetList) { _packetReceiver->handleVerifiedPacketList(std::move(packetList)); });
```
Comment (Atlante45) [inline libraries/networking/src/LimitedNodeList.cpp]: Looks like it's more than 120.

File: ice-server/src/IceServer.cpp [hunk 2]
```diff
@@ -36,7 +36,7 @@ IceServer::IceServer(int argc, char* argv[]) :
 
     // set processPacket as the verified packet callback for the udt::Socket
     using std::placeholders::_1;
```
Comment (birarda) [inline ice-server/src/IceServer.cpp]: don't need the placeholders without the bind (unless I'm missing something below that uses it)

File: libraries/networking/src/LimitedNodeList.cpp [hunk 3]
```diff
@@ -764,7 +764,7 @@ void LimitedNodeList::possiblyTimeoutSTUNAddressLookup() {
 void LimitedNodeList::addSTUNHandlerToUnfiltered() {
     // make ourselves the handler of STUN packets when they come in
     using std::placeholders::_1;
```
Comment (birarda) [inline libraries/networking/src/LimitedNodeList.cpp]: same comment on placeholders

File: tools/udt-test/src/UDTTest.cpp [hunk 4]
```diff
@@ -177,7 +177,7 @@ UDTTest::UDTTest(int& argc, char** argv) :
         // this is a receiver - in case there are ordered packets (messages) being sent to us make sure that we handle them
         // so that they can be verified
         using std::placeholders::_1;
```
Comment (birarda) [inline tools/udt-test/src/UDTTest.cpp]: same comment on placeholders

**Pull request merged**